scripts/archive-artifacts.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+#
+# Copyright 2014-2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# -------------- Import ---------------------#
+export BASE_DIR=${BASE_DIR:-"/opt/rpc-openstack"}
+source ${BASE_DIR}/scripts/functions.sh
+# ------------ End import -------------------#
+
+#---------- Required Vars -------------------#
+export BUILD_TAG=${BUILD_TAG:-""}
+export CONTAINER=${CONTAINER:-""}
+export SRC=${SRC:-""}
+#--------------- End Vars -------------------#
```

Comment:
Good call, I missed those in moving the variables, which showed a larger issue with `BUILD_TAG` in the upload script.

Latest commit removes and uses the `${WORKSPACE}/log` directory